projects/novo-examples/src/utils/drag-drop/drag-drop/drag-drop-example.html

```diff
@@ -1,19 +1,32 @@
+
 <p>Click and drag these buttons to place them in a new position. When rearranged, they emit an event to demonstrate the new order.</p>
 <div [novoDragDrop]="objects" (novoDragDropFinish)="dragFinished($event)" class="drag-box">
-    <div *ngFor="let item of objects"
+    <novo-card *ngFor="let item of objects"
         class="demo-drag-item">
-        <button class="novo-drag-target" [style.backgroundColor]="item.bgColor">{{item.btnText}}</button> <button class="remove-btn" (click)="removeObject(item)">X</button>
-        <div novoPreventDrag>{{item.name}} (Non-draggable area)</div>
-    </div>
+        <novo-card-header class="novo-drag-target {{item.bgClass}}">
+            <novo-title>{{item.headerText}}</novo-title>
+            <novo-action (click)="removeObject(item)" icon="close" tooltip="Close Item"></novo-action>
+        </novo-card-header>
+        <novo-card-content>
+            <novo-caption novoPreventDrag>{{item.name}} (Non-draggable area)</novo-caption>
+        </novo-card-content>
+    </novo-card>
 </div>
+
 <p *ngIf="objectMoved">{{objectMoved.name}} moved.</p>
-<p *ngIf="objectMoved">Current object order is: {{objectsAsString}}</p>
-<button (click)="addObject()">Add another object to list</button>
+<p>Current object order is: {{objectsAsString}}</p>
 
+<h2>Dynamic list</h2>
+<novo-button (click)="addObject2()">Add another object to list</novo-button>
 <div [novoDragDrop]="objects2" (novoDragDropFinish)="dragFinished2($event)" class="drag-box">
-    <div *ngFor="let item of objects2"
+    <novo-card *ngFor="let item of objects2"
         class="demo-drag-item">
-        <button class="novo-drag-target" [style.backgroundColor]="item.bgColor">{{item.btnText}}</button> <button class="remove-btn" (click)="removeObject(item)">X</button>
-        <div novoPreventDrag>{{item.name}} (Non-draggable area)</div>
-    </div>
+        <novo-card-header class="novo-drag-target {{item.bgClass}}">
+            <novo-title>{{item.headerText}}</novo-title>
+            <novo-action (click)="removeObject(item)" icon="close" tooltip="Close Item"></novo-action>
```

Comment:
this is calling the same removeObject function from as the demo above, which only removes items from the `objects` array populating that demo. might want to update this.

Reply:
Ah, you're right - tried this on the first list but not the second. Thanks for spotting.